apps/web/src/components/Basenames/RegistrationBackground/index.tsx

```diff
@@ -39,31 +45,77 @@ export default function RegistrationBackground() {
       </Transition>
       <Transition
         appear
-        show={isClaim || isPending}
-        className={classNames('transition-opacity', registrationTransitionDuration)}
+        show={isClaim}
+        className={classNames(
+          'transition-opacity',
+          registrationTransitionDuration,
+          grayVideoBackgroundClasses,
+        )}
+        enterFrom={classNames('opacity-0')}
+        enterTo={classNames('opacity-100')}
+        leaveFrom="opacity-100"
+        leaveTo="opacity-0"
+      >
+        {/* TODO: Lottie animation file */}
+
+        <Transition
+          appear
+          show={isClaim}
+          className={classNames(
+            'transform-gpu transition-all',
+            registrationTransitionDuration,
+            videoClasses,
+          )}
+          enterFrom="scale-0"
+          enterTo="scale-100"
+        >
+          {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
+          <video autoPlay className={videoClasses} loop>
+            <source src={globe} type="video/webm" />
```

Comment:
consider adding a fallback mp4 source (5% of global devices are on a version of ios that only has partial support for webm)
https://caniuse.com/webm

Reply:
fallback is not video (just gray or blue background) - I wanted to add mp4 but the files sizes are too large IMO (11mb for fireworks.mp4)